packages/features/eventtypes/lib/getPublicEvent.ts

```diff
@@ -150,13 +182,26 @@ export const getPublicEvent = async (
           firstUsersMetadata?.defaultBookerLayouts || defaultEventBookerLayouts
         ),
       },
+      entity: {
+        isUnpublished: unPublishedOrgUser !== undefined,
+        orgSlug: org,
+        name: unPublishedOrgUser?.organization?.name ?? null,
+      },
     };
   }
 
   const usersOrTeamQuery = isTeamEvent
     ? {
         team: {
-          slug: username,
+          OR: [
+            { slug: username },
+            {
+              metadata: {
+                path: ["requestedSlug"],
```

Comment:
Needed to make sure we also selected an unpublished organization to not end up in a 404 page